src/norm/postgres.nim

```diff
@@ -101,6 +101,20 @@ proc createTables*[T: Model](dbConn; obj: T) =
       fkGroups.add "FOREIGN KEY($#) REFERENCES $#($#)" %
         [obj.col(fld), typeof(get val.model).table, typeof(get val.model).col("id")]
 
+    when obj.dot(fld).hasCustomPragma(fk):
+      # Check val is int
+      when val is not int:
```

Comment:
I'm not sure this is idiomatic. I mean, shouldn't that be `val isNot int` or `not val is int`? Again, honest question.

Reply:
I believe from the manual that it should be ``notin``:
>   notin is the same as notIn and not_in (all-lowercase version (notin, isnot) is the preferred way of writing keywords).